### PR TITLE
Leaderboard: 1.1387 BPB — 11L LeakyReLU² + Early QAT@0.5 + GPTQ-lite + EMA

### PR DESCRIPTION
**Pull request by 0xadvait**
## Summary

11-layer 512-dim transformer with LeakyReLU(0.5)² 3x MLPs, achieving **1.1387 BPB** (3-seed mean, std 0.0005) via stride-64 sliding window evaluation.

### Key Innovation: Early QAT

Starting int6 STE fake-quantization at **LR scale < 0.5** (instead of the typical 0.15) gives ~1400 QAT steps before wallclock cutoff, reducing the post-quantization gap from 0.28 to **0.004 BPB**.

### Results (3 seeds, 8xH100 SXM)

| Seed | Sliding BPB (s64) | Artifact |
|------|-------------------|----------|
| 1337 | 1.1393 | 15.53 MB |
| 42 | 1.1386 | 15.61 MB |
| 2025 | 1.1383 | 15.52 MB |
| **Mean** | **1.1387** | |
| **Std** | **0.0005** | |

### Architecture

- 11L × 512, 8/4 GQA, 3x MLP, LeakyReLU(0.5)², U-Net skips
- EMA(0.997), Muon+AdamW WD 0.04, warmdown 3500, grad clip 0.3
- GPTQ-lite per-row clip search (5 percentiles)
- Int6 per-row for attn/MLP, int8 for embeddings, zstd-22 compression
- Train: seq_len=2048, batch=786432 tokens/step

### Run Command

```bash
SEED=1337 torchrun --standalone --nproc_per_node=8 train_gpt.py
```

### CODE REVIEW

**Comment by akhoyannh-a11y** [comment]
j hi

File: records/track_10min_16mb/2026-03-27_ByteStateTransformer/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# 11L LeakyReLU² + Early QAT@0.5 + GPTQ-lite + EMA
+
+**val_bpb: 1.1387** (sliding window stride=64, 3-seed mean) | **~15.6 MB** | 8xH100 SXM, 600s
+
+## Key Innovation: Early QAT
```
Comment (akhoyannh-a11y) [inline records/track_10min_16mb/2026-03-27_ByteStateTransformer/README.md:5]: h

**Review by akhoyannh-a11y — CHANGES_REQUESTED** [review]

**Comment by MatoTeziTanka** [comment]
### Community Review — Record: 1.1387 BPB — 11L LeakyReLU² + Early QAT@0.5 + GPTQ-lite + EMA

**BPB:** 1.1387 | **Compliance:** LOOKS CLEAN — pure-neural submission, no TTT/SLOT/n-gram-cache

**What I found in the code** (head SHA `0f2d8a620946`, file `records/track_10min_16mb/2026-03-27_ByteStateTransformer/train_gpt.py`):

Static code review found no TTT adaptation function, no SLOT optimization loop, no n-gram-cache class, and no pre-quant val-token fine-tune. The eval path uses the standard sliding-window stride-64 pattern. The submission is a pure-neural architecture iteration on the standard SP1024/SP4096/SP8192 baseline.

**CPU smoke test (CT2038 proteus-engine, 2026-04-11):** import OK in 0.03s, dim=512, layers=11, vocab=1024, code=51959 B, SMOKE_TEST_PASS

**Verdict:** LOOKS CLEAN.

**Recommendation to @cocohearts @valerio-oai @0hq @yuzhougu-oai @notapplica:** **MERGE** pending the usual record-track checks (3-seed validation, under-16MB artifact cap, ≤600s train + ≤600s eval on 8×H100 SXM). No compliance flags from the classification pass — this looks like a clean pure-neural iteration on the standard baseline.

*Auto-classification caveat:* this review was drafted by the AST-based classifier. If there's a non-standard eval mechanism (logit postprocessing, hedge mixing, etc.) that I missed because it's factored into a helper file or a non-standard function name, please flag it and I'll re-run the audit manually.

---
*Reviewed by [@MatoTeziTanka](https://github.com/MatoTeziTanka) — [The Agora](https://matotezitanka.github.io/parameter-golf). CPU smoke test (CT2038 proteus-engine, 2026-04-11): import OK in 0.03s, dim=512, layers=11, vocab=1024, code=51959 B, SMOKE_TEST_PASS. Classification via deterministic AST-based `classify_prs.py` (pattern bank derived from ~65 manually-reviewed PRs earlier in the 2026-04-11 sweep). This review was auto-drafted from a template and spot-checked before posting — if the template misread your code, please call it out so I can iterate the classifier.*
